storio-sample-app/src/main/java/com/pushtorefresh/storio/sample/db/entities/Tweet.java

```diff
@@ -14,6 +17,7 @@
 // Which will generate type mapping code in compile time,
 // You just need to link it in your code.
 @StorIOSQLiteType(table = TweetsTable.TABLE)
+@StorIOContentResolverType(uri = TweetMeta.URI_STRING)
```

Comment:
I'm afraid a lot of people will be confused by both annotations on same class, but I think I'm okay to merge it and then I'll just add separate sample app for contentresolver